### PR TITLE
Profiles

### PR DESCRIPTION
**Pull request by gcharest**
Created a data folder where we can add our professional profiles.

Next step will be to create the cards pulling the data from them.

### CODE REVIEW

**Comment by gcharest** [comment]
Linked to #502

File: _pages/en/employee-profiles.md [hunk 1]
```diff
@@ -1,23 +0,0 @@
...
-
-These profiles are to help categorize employees and determine their IT and business needs when creating a [long term](it-picture-long-term.md) and [medium term](it-picture-medium-term.md) visions as well as strategies to get there.
-
-### Profiles
-
-- Management - Manager and above
-- Power users - Development, system/database administration, analytics/reporting
-- Client services - Work directly with the public
-- General users - Usually at their desk
-- Special needs - Non-standard devices and tools
```
Comment (smellems) [inline _pages/en/employee-profiles.md]: This page is for personas of ESDC employees.  Not a profile page for the team

Comment (gcharest) [reply _pages/en/employee-profiles.md]: This page has been removed and can now be found on PR #512

Comment (smellems) [reply _pages/en/employee-profiles.md]: #512 modifies the file, but it should not be removed from in this PR.  It will cause a conflict.

Comment (gcharest) [reply _pages/en/employee-profiles.md]: Hmm, will have a look. #512 is off the master directly so yeah, I guess I should just leave it as is. Pretty sur it'll still generate a conflict in the end though.

**Pull request closed without merging**